src/session/content/chat_action_bar.rs

```diff
@@ -269,6 +293,42 @@ impl ChatActionBar {
         }
     }
 
+    pub(crate) fn handle_paste_action(&self) {
+        if let Some(chat) = self.chat() {
+            spawn(clone!(@weak self as obj => async move {
+                if let Err(e) = obj.handle_image_clipboard(chat).await {
+                    log::warn!("Error on pasting an image: {:?}", e);
+                }
+            }));
+        }
+    }
+
+    async fn handle_image_clipboard(&self, chat: Chat) -> Result<(), anyhow::Error> {
+        if let Ok((stream, mime)) = self
+            .clipboard()
+            .read_future(PHOTO_MIME_TYPES, glib::PRIORITY_DEFAULT)
+            .await
```

Comment:
This is a dilemma. With the current Flatpak permissions, we are pretty much left out in the cold here.
Otherwise, I would also not give full read permissions to the host.

Reply:
Wdym?

Reply:
Pasting an image does not work for me. It fails at this location. First, I thought the reason was that it couldn't process the file stream due to flatpak file system permissions. But as I see now the whole image is to be expected in the clipboard not just its file path. So forget about what I said in the beginning. :-P
But still cannot get this to work.

Reply:
All my 3 computers seem to be broken. But pasting does not work in Obfuscate either. Maybe this is just a Fedora bug.